neural_compressor/torch/utils/environ.py

```diff
@@ -31,20 +31,20 @@ def is_hpex_available():
     return _hpex_available
 
 
-try:
-    import intel_extension_for_pytorch as ipex
-
-    _ipex_available = True
-except:
-    _ipex_available = False
-
-
 def is_ipex_available():
```

Comment:
please don't wrap it in function, we should make env check once for all.

Reply:
Keep it under the function, since we do not want to import ipex automatically.